README.md

```diff
@@ -37,3 +37,125 @@ and JavaScript/TypeScript. In order to set up the pre-commit hooks in the JS/TS
 the `bootstrap.js` script was added. With `pre-commit` installed, running the `bootstrap.js` script is equivalent
 to `husky install`. Note that the hooks are only activated when the package is installed locally (`npm install`)
 and not when installed as a dependency.
+
+## Writing tests
+
+### Common architecture
+
+![alt text](image.png)
```

Comment:
We should explain the isolation of concerns and where cy and browser methods should be used